x/evm/keeper/querier.go

```diff
@@ -48,33 +46,12 @@ func NewQuerier(k types.BaseKeeper, n types.Nexus) sdk.Querier {
 			return QueryTokenAddressByAsset(ctx, chainKeeper, n, path[2])
 		case QTokenAddressBySymbol:
 			return QueryTokenAddressBySymbol(ctx, chainKeeper, n, path[2])
-		case QCommand:
-			return queryCommand(ctx, chainKeeper, path[2])
 		default:
 			return nil, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, fmt.Sprintf("unknown evm-bridge query endpoint: %s", path[0]))
 		}
 	}
 }
 
-func queryCommand(ctx sdk.Context, keeper types.ChainKeeper, id string) ([]byte, error) {
-	cmdID, err := types.HexToCommandID(id)
-	if err != nil {
-		return nil, sdkerrors.Wrap(types.ErrEVM, err.Error())
-	}
-
-	cmd, ok := keeper.GetCommand(ctx, cmdID)
-	if !ok {
-		return nil, sdkerrors.Wrap(types.ErrEVM, fmt.Sprintf("could not find command '%s'", id))
-	}
-
-	resp, err := GetCommandResponse(cmd)
-	if err != nil {
-		return nil, sdkerrors.Wrap(types.ErrEVM, err.Error())
-	}
-
-	return resp.Marshal()
-}
-
 // GetCommandResponse converts a Command into a CommandResponse type
 func GetCommandResponse(cmd types.Command) (types.QueryCommandResponse, error) {
```

Comment:
This method can be moved to the grpc query file

Reply:
382ce8d3407b86248665997e34f5ae03320abd88